pennylane/devices/tests/test_measurements.py

```diff
@@ -1671,7 +1691,7 @@ def test_custom_state_measurement(self, device):
         dev = device(2)
         _skip_test_for_braket(dev)
 
-        if dev.shots is not None:
+        if bool(dev.shots):
```

Comment:
```suggestion
        if dev.shots:
```
Do we need the bool? Shouldn't any value here (other than 0/None) be sufficient